projects/schematics/src/shared/constants.ts

```diff
@@ -173,6 +173,10 @@ export const PLACE_ORDER_COMPONENT = 'PlaceOrderComponent';
 export const USER_INTERESTS_SERVICE = 'UserInterestsService';
 export const USER_NOTIFICATION_PREFERENCE_SERVICE =
   'UserNotificationPreferenceService';
+export const PRODUCT_REVIEWS_COMPONENT = `ProductReviewsComponent`;
+export const SEARCH_BOX_COMPONENT = `SearchBoxComponent`;
+export const CMS_SEARCH_BOX_COMPONENT = `CmsSearchBoxComponent`;
```

Comment:
Worth to check if that would work, because it is used inside CmsComponentData type